README.md

```diff
@@ -1,5 +1,7 @@
 # cisco-cookbook
 
+[![Build Status](https://travis-ci.org/cisco/cisco-network-chef-cookbook.svg?branch=develop)](https://travis-ci.org/cisco/cisco-network-chef-cookbook)
+
```

Comment:
Yeah, looks like there are badges on http://shields.io/ for both chef cookbooks and puppet modules. That's a bit outside the scope of this PR though - mind if I add that as a separate commit to README once this is merged into develop?